### PR TITLE
Create PRWorkflow.yml

### PR DESCRIPTION
**Pull request by Ciuffi**
Notify dev channel when new PRs out for review

### CODE REVIEW

File: .github/workflows/PRWorkflow.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        SLACK_WEBHOOK_URL : ${{ secrets.SLACK_WEBHOOK_URL }}
+        PULL_REQUEST_NUMBER : ${{ github.event.pull_request.number }}
+        PULL_REQUEST_TITLE : ${{ github.event.pull_request.title }}
+        PULL_REQUEST_AUTHOR_NAME : ${{ github.event.pull_request.user.login }}
+        PULL_REQUEST_AUTHOR_ICON_URL : ${{ github.event.pull_request.user.avatar_url }}
+        PULL_REQUEST_URL : ${{ github.event.pull_request.html_url }}
+        PULL_REQUEST_BODY : ${{ github.event.pull_request.body }}
+        PULL_REQUEST_COMPARE_BRANCH_NAME : ${{ github.event.pull_request.head.ref }}
+        PULL_REQUEST_BASE_BRANCH_NAME : ${{ github.event.pull_request.base.ref }}
+        IS_SEND_HERE_MENTION : true
```
Comment (acchiang) [inline .github/workflows/PRWorkflow.yml:22]: pls do not @here for every PR going out for review thx :)

File: .github/workflows/PRWorkflow.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
```
Comment (Ciuffi) [inline .github/workflows/PRWorkflow.yml:22]:
```suggestion
        IS_SEND_HERE_MENTION : false
```

File: .github/workflows/PRWorkflow.yml (hunk 1 above)
Comment (Ciuffi) [reply .github/workflows/PRWorkflow.yml]: okay fixed

**Comment by acchiang** [comment]
LGTM 
giphy

**Pull request merged**